src/components/RadioButtonGroup/index.js

```diff
@@ -45,8 +61,62 @@ class RadioButtonGroup extends Component {
         return undefined;
     }
 
+    getCheckedOptionRef() {
+        const { value, options } = this.props;
+        const currentOptionIndex = options.findIndex(option => isOptionChecked(option, value));
+        return currentOptionIndex !== -1 ? this.optionsRefs[currentOptionIndex] : null;
+    }
```

Comment:
move this to a helper and add test to it

Reply:
this depends on component props and internal attributes...